src/datagrid/DataGrid.stories.tsx

```diff
@@ -10,8 +10,18 @@
 
 import * as React from "react";
```

Comment:
One observation. When we click on the second row (16166) select option, then all the select options get selected. I came across this after pulling in the latest code changes of this branch. I found this on firefox and chrome. Please check once on your setup.

Reply:
done.